godtools/App/Share/Data/MobileContentAuthTokenRepository/Api/MobileContentAuthTokenAPI.swift

```diff
@@ -22,13 +22,13 @@ class MobileContentAuthTokenAPI {
         self.baseURL = config.mobileContentApiBaseUrl
     }
     
-    private func getAuthTokenRequest(providerAccessToken: AuthenticationProviderAccessToken, createUser: Bool) -> URLRequest {
+    private func getAuthTokenRequest(authToken: MobileContentAuthToken, createUser: Bool) -> URLRequest {
```

Comment:
I'm a little confused about the naming now, because the `MobileContentAuthToken` struct isn't actually the auth token, it's the properties that are needed to fetch the auth token, right?

Reply:
Hey @rachaelblue that definitely is more confusing now.  I'm thinking it should be something like ```providerToken``` instead of ```providerAccessToken```.

Reply:
Right it's the properties needed to fetch the mobile-content-api auth token.

Reply:
And those properties come from the OIDC token (identity token, access token) etc.

Reply:
Maybe the struct should be ```MobileContentProviderToken``` ?